app/views/pages/faqs.html.erb

```diff
@@ -6,6 +6,270 @@
   <div class="usa-width-three-fourths usa-layout-docs-main_content">
     <h1><%= @page_title %></h1>
 
-    <%# TODO Add page content %>
+    <%# TODO Hide the checkboxes when the screen is too narrow %>
+    <div class="usa-grid" id="faq-filters">
+
+      <div class="usa-width-one-third">
```

Comment:
Why not use a (mildly) more semantic HTML5 tag here? Something like `<section>`? Note: a valid reason may be: we don't use them anywhere else and it's `div`s all the way down.